frontend/insurance/src/app/app.component.spec.ts

```diff
@@ -16,18 +20,12 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
```

Comment:
What is the point of this?

Reply:
Anytime you're expecting `componentX` to be truthy, it's basically a test for the component to be successfully created. Boilerplate provided by the CLI.